scripts/random_sampling.py

```diff
@@ -0,0 +1,244 @@
+import argparse
+import json
+import os
+from copy import deepcopy
+
+import numpy as np
+import pandas as pd
+from datasets import Dataset
+from tqdm import tqdm
+from transformers import (
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    DataCollatorWithPadding,
+    Trainer,
+    TrainingArguments,
+)
+
+from arc_tigers.eval.utils import compute_metrics
+from arc_tigers.sample.random import RandomSampler
+from arc_tigers.training.utils import get_reddit_data
+from arc_tigers.utils import load_yaml
+
+
+def imbalance_dataset(dataset, seed, class_balance):
```

Comment:
I feel like creating the imbalance should be handled elsewhere (e.g. in the data scripts)